### ISSUE TITLE
PatchDiscriminator

### ISSUE THREAD

**Issue by ck-amrahd** [nihalsid/stylegan2-ada-3d-texture#6]
Hi, Could you please upload the patch discriminator described in the paper? Also, is the final loss = image_discriminator_loss + patch_discriminator_loss? or did you use some other weighting values?
